apps/package.json

```diff
@@ -174,6 +174,7 @@
   "dependencies": {
     "filesaver.js": "0.2.0",
     "jszip": "3.0.0",
-    "query-string": "4.1.0"
+    "query-string": "4.1.0",
+    "react-addons-css-transition-group": "^15.3.0"
```

Comment:
This should prob be a dev dependency instead of a dependency (same might be true of other things listed as dependencies). Also, I'd lock to a specific version, i.e. "15.3.0" instead of "^15.3.0"